project_resource_calendar/security/calendar_security.xml

```diff
@@ -14,4 +14,53 @@
         <field name="users" eval="[(4, ref('base.user_root'))]"/>
     </record>
 
+    <record id="group_resource_calendar_editor" model="res.groups">
+        <field name="name">Resource Calendar Editor</field>
+        <field name="category_id" ref="module_category_resource_event_calendar"/>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
+    </record>
+
+    <record id="all_calendar_event_manager_access" model="ir.rule">
+        <field name="name">All Calendar Events</field>
+        <field name="model_id" ref="calendar.model_calendar_event"/>
+        <field name="domain_force">[(1,'=',1)]</field>
+        <field name="groups" eval="[(4, ref('group_resource_calendar_manager')),
+        (4, ref('group_resource_calendar_editor'))]"/>
+        <field eval="1" name="perm_unlink"/>
+        <field eval="1" name="perm_write"/>
+        <field eval="1" name="perm_read"/>
+        <field eval="1" name="perm_create"/>
+    </record>
+
+    <record id="own_calendar_event_rwcd_access" model="ir.rule">
+        <field name="name">Event at own rooms RWCD</field>
+        <field name="model_id" ref="calendar.model_calendar_event"/>
+        <field name="domain_force">[(
+            'room_id.tag_ids',
+            'in',
+            [t.id for t in  user.employee_ids.category_ids]
+            )]
+        </field>
+        <field name="groups" eval="[(4, ref('base.group_user'))]"/>
+        <field eval="1" name="perm_unlink"/>
+        <field eval="1" name="perm_write"/>
+        <field eval="1" name="perm_read"/>
+        <field eval="1" name="perm_create"/>
```

Comment:
Different than ACL. Have you tested that this works? Even if you give permission here, if permission is denied on ACL, permissions are subtracted, not added. Maybe if you give all the permissions on the ACL and you subtract here, it might have the effect that you want. 
ACL:
access_calendar_event_basic,access_calendar_event_basic,calendar.model_calendar_event,base.group_user,1,0,0,0

Reply:
We have to check this with @gaultp